packages/dotcom-server-app-context/src/validate.ts

```diff
@@ -11,6 +11,12 @@ export default function validate(field: string, value): boolean {
   if (isValid(data)) {
     return true
   } else {
-    throw Error(`Validation error: ${ajv.errorsText(isValid.errors)}, received "${value}"`)
+    throw Error(
+      `
+Validation error: ${ajv.errorsText(isValid.errors)}, received "${value}"
+
+If you want to share application specific data with the client, consider using @financial-times/dotcom-ui-data-embed.
+`.trim()
+    )
```

Comment:
Same backticks issue as above.

Reply:
When I looked at this again I noticed there was an opportunity to improve the error message so we only output the additional information when they specifically encounter the error for adding additional properties. So I've managed to address the odd formatting as well at the same time...